writing_center/appointments/__init__.py

```diff
@@ -103,6 +103,8 @@ def appointments_and_walk_ins(self):
         self.wcc.check_roles_and_route(['Tutor', 'Administrator'])
         tutor = flask_session['USERNAME']
         appointments = self.ac.get_scheduled_appointments(tutor)
+        in_progress_appointments = self.ac.get_in_progress_appointments(tutor)
+        appointments.extend(in_progress_appointments)
```

Comment:
I've never seen this before. Does it just add the lists together?

Reply:
@j-jasik, yup it just combines the lists into one list.

Reply:
list1 = [1, 2, 3]
list2 = [4, 5, 6]

list1.extend(list2) = [1, 2, 3, 4, 5, 6]
list1.append(list2) = [1, 2, 3, [4, 5, 6]]

Reply:
Nice. I'll have to log that one away to use later 👍